android/src/main/java/com/apptentive/android/sdk/reactlibrary/ApptentiveModule.kt

```diff
@@ -95,19 +95,27 @@ class ApptentiveModule(reactContext: ReactApplicationContext) : ReactContextBase
     }
   }
 
-  // Add person custom data based on key string and value of type bool, number, or string
+  // Add person custom data based on key string and value of type bool
+  // Delegated from addCustomPersonData(key, value)
   @ReactMethod
-  fun addCustomPersonData(key: String, value: Any, promise: Promise): Unit {
-    when(value) {
-      is Number -> Apptentive.addCustomPersonData(key, value)
-      is Boolean -> Apptentive.addCustomPersonData(key, value)
-      is String -> Apptentive.addCustomPersonData(key, value)
-      else -> {
-        promise.reject("Apptentive Error", "Unable to add custom person" +
-          " data for key $key: Unexpected type: ${value::class.java.simpleName}", null)
-        return
-      }
-    }
+  fun addCustomPersonDataBoolean(key: String, value: Boolean, promise: Promise): Unit {
+    Apptentive.addCustomPersonData(key, value)
+    promise.resolve(true)
+  }
+
+  // Add person custom data based on key string and value of type double
+  // Delegated from addCustomPersonData(key, value)
+  @ReactMethod
```

Comment:
Should this be Double or Number?

Reply:
The @ReactMethod annotation does not support the Number type as an argument so I chose Double to be the next best type to represent numbers